docs/Lifecycle/index.md

```diff
@@ -0,0 +1,257 @@
+
+Each one of them might be executed in parallel or sequentially. However for each proposal there is a streamlined development process, outlined on following picture.<br>
+![Development streamline](assets/development-streamline.png "Development streamline")<br>
+
+## **Roles**
+SDL lifecycle considers interactions of following groups:
+
+- Stakeholders (primary responsible role SDL Project Manager)
+- [Change Control Board (CCB)][CCB-LINK] (primary responsible role CCB Group Manager)
+- Maintainers (primary responsible role is **Task Assignee**)
+- Community (primary responsible role for propsal review is **Proposal Author**)
+
+### Stakeholders
+Primary owners of SDL.
+Group responsbilities:
+1. Assign responcible for proposal processing
+2. Assign responsible for tasks realisation (*Maintainers*)
+3. Approve potential contributors
+4. Define priorities/business value of approved proposals
+5. Delegate, by need, responcibility 1. and/or 2. to [CCB][CCB-LINK]
+
+### Change Control Board
+[CCB][CCB-LINK] is group of technical specialists gathered and approved by Stakeholders.
+Their primary respocibility is to control state of the product.
+Detailed description on group duties available in [here][CCB-LINK].
+
+### Maintainers
+Approved by Stakeholders potential contributors.
+Their primary responsibility is providing delivery according SDL Contribution or SDL Release process.
+For each task contributor must be assigned by Stakeholder or CCB.
+Relations between Stakeholders and MAintainers are not part of this description.
+
+### Community
+Consumers of new SDL releases.
+Generators of ideas and proposals for improvement in SDL project.
+People which delivery most of the innovations in SDL through SDL Evolution process.
+
+## **Processes**
+This chapter alines important, for product quality, activities in a production chain.
+All processes are built around GitHub and therefore presented as set of GitHub issues and wiki pages.
+
+### [SDL Evolution][SDL-EP-LINK]
+Change managemenet is important part of production chain.<br>
+*Community* should propose improvements in the scope of [SDL Evolution][SDL-EP-LINK] process.<br>
+Other groups which are involved in this process: *Stakeholders* and *[CCB][CCB-LINK]*.<br>
+Usually Stakeholders define responcible for proposal processing, and [CCB][CCB-LINK] helps in clarifcation of specifications and impact.<br>
+For proposals which recieved status **Accepted** after [review][SDL-EP-LINK] respective *proposal statement of work* will be deliniated.<br>
+Statemenet of work for proposal includes set of GitHub issues created in repositories of affected *parts of product*.<br>
+
+> **NOTE:** [SDL Evolution][SDL-EP-LINK] process stipulates possible communication channels,
+> through which *Community* can delivery intial requests or answer questions.
+
+**Outcome:** 
+- Issue in [SDL Evolution repository][SDL-EP-LINK] for approved proposal (mandatory for accepted proposals only)
```

Comment:
Could we reorder list in more easy-look way
* **mandatory** - Do1
* **mandatory** - Do2
* **mandatory** - Do3
* *optional* - Do4